lisa/sut_orchestrator/baremetal/features.py

```diff
@@ -41,3 +41,29 @@ class SerialConsole(ClusterFeature):
     def _get_inner_type(self) -> Type[Feature]:
         platform: BareMetalPlatform = self._platform  # type: ignore
         return platform.cluster.get_serial_console()
+
+
+class SecurityProfile(features.SecurityProfile):
```

Comment:
Is it still needed?

Reply:
If so, the declaration in cluster should create an instance of this class, not the root one.